doc/release_notes.rst

```diff
@@ -12,8 +12,8 @@ embedded development through an open source platform. Check out the
 :ref:`introduction` for more information.
 
 The project ACRN reference code can be found on GitHub in
-https://github.com/projectacrn.  It includes the ACRN hypervisor, the
-ACRN device model, and documentation.
+`https://github.com/projectacrn/acrn-hypervisor`_.  It includes the
```

Comment:
This would generate an error (if doc building were part of our CI process). The syntax you're using expects a target with this name to be defined (e.g., 

    _https://github.com/projectacrn/acrn-hypervisor:  xxx

The correct syntax here is simply:

    https://github.com/projectacrn/acrn-hypervisior 

without any of the backquotes or underscore.